system/jlib/jsocket.hpp

```diff
@@ -368,8 +368,10 @@ class jlib_decl ISocket : extends IInterface
     virtual bool join_multicast_group(SocketEndpoint &ep) = 0;  // for udp multicast
     virtual bool leave_multicast_group(SocketEndpoint &ep) = 0; // for udp multicast
 
+    virtual void set_ttl(unsigned _ttl) =0;  // set TTL
+
     virtual size32_t get_receive_buffer_size() =0;              // get OS receive buffer
-    virtual void set_receive_buffer_size(size32_t sz) =0;           // set OS receive buffer size
+    virtual void set_receive_buffer_size(size32_t sz) =0;       // set OS receive buffer size
```

Comment:
If you are going to fix the formatting, fix the missing space between = and 0 too?